requirements.txt

```diff
@@ -2,16 +2,16 @@ sarif-om
 jschema_to_python
 defusedxml
 gitpython
-shiftleft-scan-reports
+git+https://github.com/perrito666/scan-reports.git
```

Comment:
That's going to be there permanently?

Reply:
`scan-reports` repo wasn't public. It is now, though (http://github.com/ShiftLeftSecurity/scan-reports).

Reply:
We also own the entry in PyPI.

Reply:
Nope, did you merge my changes?